src/Cli/ProjectCommands.php

```diff
@@ -531,6 +531,153 @@ public function projectUpstreamUpdate($remote, $options = ['as' => 'default', 'p
+        $last_badge_line = -1;
+        $badge_insert_line = -1;
+        $empty_line_after = true;
+        foreach ($readme_lines as $line_number => $line) {
+            if ($first_empty_line == -1 && empty(trim($line))) {
+                $first_empty_line = $line_number;
+            }
+            // Is this line a badge?
+            if (preg_match('/\[\!\[[A-Za-z0-9\s]+\]\(.*\)/', $line)) {
+                $last_badge_line = $line_number;
+                // Is this line the License badge?
+                if (preg_match('/\[\!\[License]\(.*\)/', $line)) {
+                    if ($line_number) {
+                        $badge_insert_line = $line_number;
+                    } else {
+                        $badge_insert_line = 0;
+                    }
+                    $empty_line_after = false;
+                }
+            } else {
+                if ($last_badge_line != -1) {
+                    // We already found the badges, exit foreach.
+                    break;
+                } elseif ($line_number > $number_of_lines_to_search) {
+                    // We've searched enough lines, exit foreach.
+                    break;
+                }
+            }
+        }
+        if ($badge_insert_line === -1) {
+            if ($last_badge_line !== -1) {
+                // If we found badges, we'll insert this badge after the last badge.
+                $badge_insert_line = $last_badge_line + 1;
+            } elseif ($first_empty_line !== -1) {
+                // If we didn't find any badges, we'll insert this badge at the first empty line.
+                $badge_insert_line = $first_empty_line + 1;
+            } else {
+                // Final fallback: insert badge in the second line of the file.
+                $badge_insert_line = 1;
+            }
+        }
+        return [$badge_insert_line, $empty_line_after];
+    }
+
+    /**
+     * Get right badge markdown.
+     */
+    protected function getSupportLevelBadge($level)
+    {
+        $badges = [
+            'ea' => '[![Early Access](https://img.shields.io/badge/pantheon-EARLY_ACCESS-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/early-access?q=org%3Apantheon-systems)',
+            'la' => '[![Limited Availability](https://img.shields.io/badge/pantheon-LIMITED_AVAILABILTY-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/limited-availability?q=org%3Apantheon-systems)',
+            'actively-supported' => '[![Actively Maintained](https://img.shields.io/badge/pantheon-actively_maintained-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/actively-maintained?q=org%3Apantheon-systems)',
+            'minimally-supported' => '[![Minimal Support](https://img.shields.io/badge/pantheon-minimal_support-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/minimal-support?q=org%3Apantheon-systems)',
```

Comment:
Maybe change the shortcuts to just `active` and `minimal` to keep the support column width down in the csv file.